keyboards/claw44/readme.md

```diff
@@ -0,0 +1,14 @@
+# Claw44
+
+![Claw44](https://i.imgur.com/5a8iogl.jpg)
+
+A split keyboard with 3x6 vertically staggered keys and 4 thumb keys.
+
+Keyboard Maintainer: [@yfuku_](https://twitter.com/yfuku_) 
```

Comment:
Keyboard information here doesn't specify the supported hardware, or where to get it.

Line 7 will need another space at the end if that gets added.

```suggestion
Keyboard Maintainer: [@yfuku_](https://twitter.com/yfuku_)  
```